### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dirien**
Add `upstream-provider-org` to Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,20 +53,38 @@ Usage:
...
 
 Flags:
+      --allow-missing-docs              If true, don't error on missing docs during tfgen.
+                                        This is equivalent to setting PULUMI_MISSING_DOCS_ERROR=${! VALUE}.
       --experimental                    Enable experimental features, such as auto token mapping and auto aliasing
   -h, --help                            help for upgrade-provider
+      --java-version string             The version of pulumi-java-gen to target.
       --kind strings                    The kind of upgrade to perform:
-                                        - "all":     Upgrade the upstream provider and the bridge. Shorthand for "bridge,provider,code".
+                                        - "all":     Upgrade the upstream provider and the bridge. Shorthand for "bridge,provider,code,pf".
```
Comment (ericrudder) [inline README.md:62]: should we alphabetize list (both in the "shorthand" list AND the entire --kind list?)

Comment (dirien) [reply README.md:62]: @iwahbe should I change the code in this PR for this or do you prefer an extra PR?

**Review by guineveresaenger — APPROVED** [review]
Thank you for this update Engin!

The changes look good to me as-is. We can alphabetize separately.

File: README.md (hunk 1 above)
Comment (iwahbe) [reply README.md:62]: We can do it in a follow up PR.

**Pull request merged**